message_definitions/v1.0/ardupilotmega.xml

```diff
@@ -31,7 +31,7 @@
       </entry>
       <entry value="212" name="MAV_CMD_DO_AUTOTUNE_ENABLE">
         <description>Enable/disable autotune.</description>
-        <param index="1">Enable (1: enable, 0:disable).</param>
+        <param index="1" label="Enable" minValue="0" maxValue="1" increment="1">Enable (1: enable, 0:disable).</param>
```

Comment:
Several of these seem to be indicating we're imposing a type on these - in this case "boolean" by specifying min/max and increment... `Number` is being constrained to be a positive integer, that sort of thing.

A GCS could infer a type from the bits we're giving it, but perhaps we could be more direct?

Reply:
Yes, that is exactly what we're doing. 

Perhaps we should define an enum with values enable/disable in common? Or were you thinking of something different?

Reply:
@peterbarker FYI ^^^